ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -919,56 +910,66 @@ static void ompi_osc_rdma_ensure_local_add_procs (void)
  * @return OMPI_SUCCESS if BTLs can be found
  * @return OMPI_ERR_UNREACH if no BTLs can be found that match
  *
- * In this case an "alternate" BTL is a BTL that does not provide true RDMA but
- * can use active messages using the BTL base AM RDMA/atomics. Since more than
- * one BTL may be needed for this support the OSC component will disable the
- * use of registration-based RDMA (these BTLs will not be used) and will use
- * any remaining BTL. By default the BTLs used will be tcp and sm but any single
- * (or pair) of BTLs may be used.
+ * This function is used when there ompi_osc_rdm_query_btls() failed to find
```

Comment:
failed to find a single btl that can communicate with all peers and supports remote completion.

Reply:
Thank you! fixed in new revision.